Frontend/src/pages/admin/AdminStudentsPage.vue

```diff
@@ -37,8 +37,16 @@
           <q-table
             :data="students"
             :columns="columns"
-            row-key="studentName"
+            row-key="email"
             @row-click="goToStudentCoop"
+            selection="multiple"
+            :selected.sync="selected"
+          />
+          <q-btn
+            class="dashBtn"
+            label="Send Notification to Selected Students"
+            color="primary"
+            @click="sendNotif"
```

Comment:
Does the admin no longer have a page where they can view students and do filtering?

Reply:
this is that page
??

Reply:
Wait yeah nevermind lol

Reply:
whats the point of filtering if ur not gonna send them a notification or output the list in someway